lib/src/view/broadcast/broadcast_boards_tab.dart

```diff
@@ -60,19 +60,16 @@ class BroadcastBoardsTab extends ConsumerWidget {
             : BroadcastPreview(
                 games: value.games.values.toIList(),
                 roundId: roundId,
-                broadcastTitle: broadcastTitle,
-                roundTitle: value.round.name,
+                title: value.round.name,
+                tournamentSlug: tournamentSlug,
+                roundSlug: value.round.slug,
               ),
         AsyncError(:final error) => SliverFillRemaining(
             child: Center(
               child: Text('Could not load broadcast: $error'),
             ),
           ),
-        _ => const SliverFillRemaining(
-            child: Center(
-              child: CircularProgressIndicator.adaptive(),
-            ),
-          ),
+        _ => const BroadcastPreview.loading()
```

Comment:
If there is a way to know in advance the number of boards, then we can use the shimmer. Otherwise I think it is ugly to see a flash of 10 shimmer boards where in fact only one is show eventually.

Reply:
Having only one board in a round is pretty rare. In the majority of cases I think the board shimmer looks nicer than the circular indicator.

Reply:
Unfortunately the world championship is the most watch broadcast... even if it is not the majority it will be seen much more times than other tournaments with more boards.

Reply:
It does unless there is only one or two board and you see a flash of shimmer if the request completes very quickly.